identity/app/idapiclient/UserUpdateDTO.scala

```diff
@@ -11,5 +17,6 @@ case class UserUpdate (
   password: Option[String] = None,
   userGroups: Option[Set[GroupMembership]] = None,
   socialLinks: Option[Set[SocialLink]] = None,
-  adData: Option[Map[String, AnyRef]] = None
-);
+  adData: Option[Map[String, AnyRef]] = None,
+  consents: Option[List[Consent]] = None
```

Comment:
Are `statusFields` still needed in this now that we have `consents`?

Reply:
Removed.